.github/workflows/t2_reviews.yaml

```diff
@@ -0,0 +1,50 @@
+# Github Action to Automatically assign T2 Reviewers to framework touched PRs
+
+name: Auto assign T2 Reviewers
+
+on:
+  pull_request:
+    types: ["opened", "synchronize", "reopened"]
+    paths:
+      - '*'
```

Comment:
Does this catch all of the files in the root dir?

Reply:
Right!